tensorflow/core/kernels/relu_op.cc

```diff
@@ -50,60 +50,81 @@ typedef Eigen::SyclDevice SYCLDevice;
+                                                                                \
+  template <>                                                                   \
+  void ReluGrad<GPUDevice, T>::operator()(                                      \
+      const GPUDevice& d, typename TTypes<T>::ConstTensor gradients,            \
+      typename TTypes<T>::ConstTensor features,                                 \
+      typename TTypes<T>::Tensor backprops);                                    \
+  extern template struct ReluGrad<GPUDevice, T>;                                \
+                                                                                \
+  template <>                                                                   \
+  void Relu6<GPUDevice, T>::operator()(                                         \
+      const GPUDevice& d, typename TTypes<T>::ConstTensor features,             \
+      typename TTypes<T>::Tensor activations);                                  \
+  extern template struct Relu6<GPUDevice, T>;                                   \
+                                                                                \
+  template <>                                                                   \
+  void Relu6Grad<GPUDevice, T>::operator()(                                     \
+      const GPUDevice& d, typename TTypes<T>::ConstTensor gradients,            \
+      typename TTypes<T>::ConstTensor features,                                 \
+      typename TTypes<T>::Tensor backprops);                                    \
+  extern template struct Relu6Grad<GPUDevice, T>;                               \
+                                                                                \
+  template <>                                                                   \
+  void Elu<GPUDevice, T>::operator()(const GPUDevice& d,                        \
+                                     typename TTypes<T>::ConstTensor features,  \
+                                     typename TTypes<T>::Tensor activations);   \
+  extern template struct Elu<GPUDevice, T>;                                     \
+                                                                                \
+  template <>                                                                   \
+  void EluGrad<GPUDevice, T>::operator()(                                       \
+      const GPUDevice& d, typename TTypes<T>::ConstTensor gradients,            \
+      typename TTypes<T>::ConstTensor activations,                              \
+      typename TTypes<T>::Tensor backprops);                                    \
+  extern template struct EluGrad<GPUDevice, T>;                                 \
+                                                                                \
+  template <>                                                                   \
+  void Selu<GPUDevice, T>::operator()(const GPUDevice& d,                       \
```

Comment:
If `const GPUDevice& d` is moved to the next line with 4 spaces indent, then the above lines after `#define DECLARE_GPU_SPEC(T)` wouldn't have to be changed.